README.md

```diff
@@ -36,6 +48,14 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. A watch connect to the cloud that supports document storage and messaging. Different groups have different permissions for transferring files to one another. File can be transferred with a note through proximity or tapping the user's id and once the file has transferred the watch vibrates and you receive a text or emoji.
+  * Weakness
```

Comment:
Good! I instinctively thought of this as the logical weakness as well. One also has to consider how the problem you were given by the other group is being solved using current file sharing methods.